docs/reference/components/sinks/datadog_logs.cue

```diff
@@ -3,6 +3,13 @@ package metadata
 components: sinks: datadog_logs: {
 	title: "Datadog Logs"
 
+	description: """
+		Sends data to Datadog. The Datadog platform can leverage specifics fields (namely `ddtags`, `ddsource` and `service`) to further categorize
+		and filter logs. For further details on the purpose of those fields please check the [official Datadog documentation](\urls.datadog_tags).
+		The source (`ddsource` field) and service (`service` field) are expected to be plain text value. The list of tags (The `ddtags` field) should
+		be a single string of coma separated tags, e.g. `tag1:val1,tag2:val2,tag3:val3`.
```

Comment:
The last 2 sentences that describe the schema should probably be represented as structured data in the `input` field, agree? Currently the `input` field only accepts booleans on the types, but we should expand that to specify special fields like these.

cc @lucperkins and @jszwedko for thoughts.

Reply:
As we do have #MetricInput/#MetricOutput & #LogOutput (as per https://github.com/timberio/vector/blob/master/docs/reference/components.cue#L402-L424). I would say we could imagine having #LogInput (or rename & reuse #LogOutput if there is no difference) to describe usable fields for sinks that can more or less strict requirement (e.g. https://github.com/timberio/vector/pull/7678/files#diff-90d1289b0a2da6889d20d7d009dcefcac4cf814c42158aa6ffafaa48d8e4b6d4R159-R172 would benefit from precise fields documentation). WDYT ? cc @lucperkins @jszwedko

Reply:
👍 that seems reasonable. It seems like we'd just add a `fields: #Schema` to `#LogInput`. We would need to make sure we still model the fact that events can have arbitrary fields outside of the special ones.